seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/Transaction.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.file.sink.writer;
+
+import org.apache.seatunnel.connectors.seatunnel.file.sink.commit.FileCommitInfo2;
+import org.apache.seatunnel.connectors.seatunnel.file.sink.state.FileSinkState2;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Optional;
+
+public interface Transaction extends Serializable {
```

Comment:
Some method be deleted, Can you explain why they were deleted?

```
/**
     * A new transaction needs to be started after each checkpoint is completed.
     *
     * @param checkpointId A checkpoint indicates that all tasks have a status snapshot operation
     * @return transactionId
     */
    String beginTransaction(@NonNull Long checkpointId);

    /**
     * Abort current Transaction, called when {@link org.apache.seatunnel.connectors.seatunnel.file.sink.TransactionStateFileSinkWriter#prepareCommit()} or {@link org.apache.seatunnel.connectors.seatunnel.file.sink.TransactionStateFileSinkWriter#snapshotState(long)} failed
     */
    void abortTransaction();

    /**
     * Get all transactionIds after the @param transactionId
     * This method called when {@link AbstractFileSink#restoreWriter(SinkWriter.Context, List)}
     * We get the transactionId of the last successful commit from {@link FileSinkState} and
     * then all transactionIds after this transactionId is dirty transactions that need to be rollback.
     *
     * @param transactionId The transactionId of the last successful commit get from {@link FileSinkState}
     * @return transactionId list
     */
    List<String> getTransactionAfter(@NonNull String transactionId);


    /**
     * rollback the transaction which is not be commit
     *
     * @param transactionIds transactionIds
     */
    void abortTransactions(List<String> transactionIds);

```

Reply:
The functions of these methods are all integrated into the committer and transaction

Reply:
> The functions of these methods are all integrated into the committer and transaction

Sorry, I don't found the `getTransactionAfter` method in this pr.

Reply:
Resolved.